pages/en/network-transition-faq.mdx

```diff
@@ -0,0 +1,236 @@
+---
+title: Network Transition FAQ
+---
+
+Developers will have plenty of time to migrate their subgraphs to the decentralized network. Exact timelines will vary from chain to chain based on Indexer and network readiness-the hosted service will not end support for all chains at once and will not be sunset abruptly.
+
+Each chain on the hosted service, including Ethereum, will sunset gradually as it is supported on the decentralized network to achieve feature parity and a high quality of service. This will happen on a chain-to-chain basis with help from Indexers in the [MIPs program](https://thegraph.com/blog/mips-multi-chain-indexing-incentivized-program/), to enable full support for each chain on the decentralized network.
+
+To add more clarity around continued support for each chain on the hosted service, these FAQs answer common questions regarding the specifics of the network transition process. If you would like to start the subgraph migration process now, here is a [step-by-step guide](https://thegraph.com/blog/how-to-migrate-ethereum-subgraph). To skip to the migration FAQ, [click here](#migration-fa-qs-answers-to-common-questions).
```

Comment:
Huh, the auto-generated `migration-fa-qs-answers-to-common-questions` slug is weird, but nothing we can do about it now. 😅

Reply:
Wow, nice catch! Didn't even notice.

Reply:
Don't worry about it, it's an issue with the package we're using. I guess it has to do with the fact that `FAQ` is an acronym.